predicators/approaches/active_sampler_learning_approach.py

```diff
@@ -313,7 +315,11 @@ def _learn_wrapped_samplers(self,
         for old_nsrt in self._nsrts:
             if old_nsrt.option not in new_nsrt_options:
                 new_test_nsrts.add(old_nsrt)
-                self._nsrt_to_explorer_sampler[old_nsrt] = old_nsrt._sampler  # pylint: disable=protected-access
+                self._nsrt_to_explorer_sampler[
+                    old_nsrt] = _wrap_sampler_exploration(
+                        old_nsrt._sampler,  # pylint: disable=protected-access
+                        lambda o, _, params: [1.0] * len(params),
```

Comment:
I think I understand what's happening here now but I was initially confused -- can you add a comment?